src/Screencast/main.vala

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env -S vala workbench.vala --pkg libadwaita-1 --pkg libportal-gtk4 --pkg gstreamer-1.0
+
+private Gtk.Picture output;
+
+async void start_screencast_session (Xdp.Portal portal, Xdp.Parent parent) {
+    try {
+        Xdp.Session session = yield portal.create_screencast_session (Xdp.OutputType.MONITOR,
+            Xdp.ScreencastFlags.NONE,
+            Xdp.CursorMode.EMBEDDED,
+            Xdp.PersistMode.TRANSIENT,
+            null,
+            null);
```

Comment:
Vala has a very nice feature which allows you, in many cases, to skip the namespace and enum name so you don't have to type a lot. In this case:
```suggestion
        Xdp.Session session = yield portal.create_screencast_session (MONITOR,
            NONE,
            EMBEDDED,
            TRANSIENT,
            null,
            null);
```

Which is preferred. A small description right next to the parameters would be nice too.